### DISCUSSION TITLE
Wazuh o365 integration problems

### DISCUSSION THREAD

**Discussion by benjaminsuljic** [wazuh/wazuh#17640] [General]
Hi all, 

I have been following this tutorial in order to integrate Wazuh and o365. 
https://documentation.wazuh.com/current/office365/monitoring-office365-activity.html#wazuh-configuration

As soon as I modified ossec.conf file and added bellow code snippet inside <ossec_config> </ossec_config>
```
<office365>
    <enabled>yes</enabled>
    <interval>1m</interval>
    <curl_max_size>1M</curl_max_size>
    <only_future_events>yes</only_future_events>
    <api_auth>
        <tenant_id>your_tenant_id</tenant_id>
        <client_id>your_client_id</client_id>
        <client_secret>your_client_secret</client_secret>
    </api_auth>
    <subscriptions>
        <subscription>Audit.SharePoint</subscription>
    </subscriptions>
</office365>
```

My wazuh server stopped working. I have reverted to backup configuration but still I get the message "Wazuh dashboard server is not ready yet" when I try to open Wazuh dashboard. Any help is appreciated.

**Comment by mdiego92** [comment]
Hello @benjaminsuljic,

If you reverted the changes, and you're still having an issue trying to start Wazuh Dashboard it could mean it is not related.
Could you please confirm you're still having this issue and share with us the output of the following commands?

```
cat /var/ossec/logs/ossec.log | grep -iE 'ERR|WARN|CRIT
journalctl -u wazuh-dashboard --no-pager
```

Sorry for the late response. Looking forward to your feedback!

**Comment by benjaminsuljic** [comment]
Hi Diego,

Thank you for responding. As you stated in your email I had another change
made by my college unrelated to the one that I wrote. So all is corrected.
Wazuh is running fine.

Benjamin

On Fri, Jul 14, 2023 at 11:40 AM Diego Mendez Sakugawa <
***@***.***> wrote:

> Hello @benjaminsuljic <https://github.com/benjaminsuljic>,
>
> If you reverted the changes, and you're still having an issue trying to
> start Wazuh Dashboard it could mean it is not related.
> Could you please confirm you're still having this issue and share with us
> the output of the following commands?
>
> cat /var/ossec/logs/ossec.log | grep -iE 'ERR|WARN|CRIT
> journalctl -u wazuh-dashboard --no-pager
>
> Sorry for the late response. Looking forward to your feedback!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wazuh/wazuh/discussions/17640#discussioncomment-6448076>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2SHKIVDRK6LZ6TZA4RA4ZDXQEHYDANCNFSM6AAAAAAZRO5XFM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
mr.sc. Benjamin Suljić, dipl.ing.el
email:
***@***.*** ***@***.***>*

*https://www.linkedin.com/in/benjaminsuljic/
<https://www.linkedin.com/in/benjaminsuljic/> *
